packages/printable-itinerary/src/access-leg.js

```diff
@@ -2,15 +2,24 @@ import coreUtils from "@opentripplanner/core-utils";
 import { humanizeDistanceString } from "@opentripplanner/humanize-distance";
 import PropTypes from "prop-types";
 import React from "react";
+import AccessibilityAnnotation from "./accessibility-annotation";
 
 import * as S from "./styled";
 
-export default function AccessLeg({ config, leg, LegIcon }) {
+export default function AccessLeg({
+  accessibilityScoreGradationMap,
+  config,
+  leg,
+  LegIcon
+}) {
   return (
     <S.Leg>
-      <S.ModeIcon>
-        <LegIcon leg={leg} />
-      </S.ModeIcon>
+      <AccessibilityAnnotation
```

Comment:
The accessibility annotation is too large for the print view IMO. It looks larger than the mode icon itself! I would have the width at 40px maximum for this view.

Reply:
Have attempted to resolve this with 9b052113f918c4d8c638f99fb623ad71b8bba15d which removes the width constraint and instead uses height to make the entire label smaller.